kernel/term.mli

```diff
@@ -40,6 +40,27 @@ val subterm : term -> position -> term
 (** [subterm t p] returns the subterm of [t] at position [p].
     Raises InvalidSubterm in case of invalid position in given term. *)
 
+type 'a comparator = 'a -> 'a -> int
+(** Type for comparison functions *)
+
+val compare_term : name comparator -> term comparator
```

Comment:
Can you comment whether this order have been chosen randomly or whether there is a reason? In particular, is this order may have an impact on the performances?